{
  "comments": [
    {
      "key": {
        "uuid": "0c85f228_ac12cb7a",
        "filename": "tools/test/tools_helper_unittest.cpp",
        "patchSetId": 5
      },
      "lineNbr": 53,
      "author": {
        "id": 1000388
      },
      "writtenOn": "2021-04-06T16:54:04Z",
      "side": 1,
      "message": "super minor nit: could you cover the case here or in the other test that open_read | committed would also pass (as expected?)",
      "range": {
        "startLine": 53,
        "startChar": 0,
        "endLine": 53,
        "endChar": 59
      },
      "revId": "d205844f49d18df2613011c8fbdfd469829bdc14",
      "serverId": "adbd0a64-1f21-4d83-b585-671fe73cb6e4",
      "unresolved": true
    }
  ]
}